pyscitt/pyscitt/cli/retrieve_signed_claims.py

```diff
@@ -52,11 +53,22 @@ def cli(fn):
         type=Path,
         help="Folder containing JSON parameter files of SCITT services to trust",
     )
+    parser.add_argument(
+        "--dont-embed-receipt",
```

Comment:
```suggestion
        "--include-receipt",
```
Avoiding double negative

Reply:
See https://github.com/microsoft/scitt-ccf-ledger/pull/196#discussion_r1678073658